core-build/gulp-core-build/src/tasks/GulpTask.ts

```diff
@@ -145,7 +144,7 @@ export abstract class GulpTask<TTaskConfig> implements IExecutable {
    * @returns a Promise, a Stream or undefined if completeCallback() is called
    */
   public abstract executeTask(
-    gulp: gulp.Gulp | GulpProxy,
```

Comment:
>p: gulp.Gulp | GulpProxy, [](start = 7, length = 25)

Are you removing GulpProxy?

Reply:
just missed this change

---
In reply to: [159797130](https://github.com/Microsoft/web-build-tools/pull/383#discussion_r159797130) [](ancestors = 159797130)